functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/junit5/LocalChangesTestsSelectionExecutionFunctionalTest.java

```diff
@@ -0,0 +1,45 @@
+package org.arquillian.smart.testing.ftest.junit5;
+
+import java.util.Collection;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.arquillian.smart.testing.ftest.testbed.project.TestResults;
+import org.arquillian.smart.testing.ftest.testbed.testresults.TestResult;
+import org.arquillian.smart.testing.rules.TestBed;
+import org.arquillian.smart.testing.rules.git.GitClone;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.SELECTING;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Strategy.CHANGED;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LocalChangesTestsSelectionExecutionFunctionalTest {
+
+    @ClassRule
+    public static final GitClone GIT_CLONE = new GitClone("https://github.com/lordofthejars/junit5-demo.git");
+
```

Comment:
Are we going to use the same repository or we are going to transfer ownership to arquillian org?

Reply:
Yes, I am not sure also how we should named in arquillian, because it is not really a dogfooding project. @bartoszmajsak WDYT about this?

My point now is that this is a really small project just to check that the file creation and read is done correctly, which is enough in case of JUnit5. So maybe call it `smart-testing-junit5-demo` in arquillian organization is enough.

Reply:
Yes, this is rather a demo project which we can transfer to the arq organization. We only do thin smoke test to be sure integration is fine, as the core logic which are tests are focusing on are not the test execution (so junit4 or junit5), but selection logic so our provider which delegates to aforementioned runners.

Let's move it as @lordofthejars suggests.